src/InjectHostDevBufferCopies.cpp

```diff
@@ -332,7 +333,11 @@ class InjectBufferCopiesForSingleBuffer : public IRMutator2 {
     // leaf.
 
     Stmt visit(const For *op) override {
-        // All copies happen at the same loop level as the allocation
+        // All copies happen at the same loop level as the allocation (TODO: This prevents hoisting allocations)
```

Comment:
@abadams Do you mind weighing in on this?

Reply:
I have no memory of this place...

I can see that split store_at compute_at might need copies to happen at the compute_at level instead of the allocation level, but that's just not what the model is. Maybe just revert this todo.